plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
Well, they often release a new driver for new game. I don't expect people to upgrade their driver every weeks. But code was working with a 3 months driver. Latest driver contains the full fix but it doesn't seem to be mandatory. GIT version aren't stable neither supported version so if they want to keep an older driver longer they can use the stable 1.4.0 or an older git.

It is impossible to support a long range of driver version. Potentially I could do it here, but there will be others issues. Even if it is working now, it could break again for a future change.

Please note that legacy drivers (and potentially a bit newer/older) are still blacklisted.